size: add page (#8604)
tldr-pages/tldr: pages/linux/size.md

# size

> Displays the sizes of sections inside binary files.

> > More information: <https://www.ibm.com/docs/en/aix/7.2?topic=s-size-command>.

Reviewer: afaik aix is not linux
```suggestion
> More information: <https://sourceware.org/binutils/docs/binutils/size.html>.
```

> - Display the size of sections of object or executable file:

Reviewer: ```suggestion
- Display the size of sections in a given object or executable file:
```

> `size {{file}}`

Reviewer: ```suggestion
`size {{path/to/file}}`
```

> - To display the size of an executable file in [o]ctal:
> 
> `size -o {{file}}`

Reviewer: ```suggestion
- To display the size of sections in a given object or executable file in [o]ctal:

`size {{-o|--radix=8}} {{path/to/file}}`
```

> - To display the size of sections in a given object or executable file in [o]ctal:

Reviewer 2: ```suggestion
- Display the size of sections in a given object or executable file in [o]ctal:
```

> - To display the size of an executable file in [d]ecimal:
> 
> `size -d {{file}}`

Reviewer: ```suggestion
- To display the size of sections in a given object or executable file in [d]ecimal:

`size {{-d|--radix=10}} {{path/to/file}}`
```

> - To display the size of sections in a given object or executable file in [d]ecimal:

Reviewer 2: ```suggestion
- Display the size of sections in a given object or executable file in [d]ecimal:
```

> - To display the size of an executable file in he[x]adecimal:
> 
> `size -x {{file}}`

Reviewer: ```suggestion
- To display the size of sections in a given object or executable file in he[x]adecimal:

`size {{-x|--radix=16}} {{path/to/file}}`
```

> - To display the size of sections in a given object or executable file in he[x]adecimal:

Reviewer 2: ```suggestion
- Display the size of sections in a given object or executable file in he[x]adecimal:
```
